bot/cogs/tags.py

```diff
@@ -160,12 +171,12 @@ def _command_on_cooldown(tag_name: str) -> bool:
                 )
                 return True
 
-        return False
+        return True
 
     @tags_group.command(name='get', aliases=('show', 'g'))
-    async def get_command(self, ctx: Context, *, tag_name: TagNameConverter = None) -> None:
-        """Get a specified tag, or a list of all tags if no tag is specified."""
-        await self._get_command(ctx, tag_name)
+    async def _get_command(self, ctx: Context, *, tag_name: TagNameConverter = None) -> None:
```

Comment:
This name could have just been left alone since, as I mentioned, `get_commands` is not an appropriate name for the other function anyway.